core/src/main/java/edu/wpi/grip/core/operations/network/http/DataHandler.java

```diff
@@ -69,6 +71,7 @@
   }
 
   @Override
+  @SuppressFBWarnings("UW_UNCOND_WAIT") // Bug in FindBugs. There is a condition.
```

Comment:
There's actually a field in the warning that you can fill out called "justification"
http://findbugs.sourceforge.net/api/edu/umd/cs/findbugs/annotations/SuppressFBWarnings.html#justification()
So you can do `@SuppressFBWarnings("UW_UNCOND_WAIT", justification = "Bug in FindBugs. There is a condition.")